builtin/plugins/dposv3/config.go

```diff
@@ -0,0 +1,30 @@
+package dposv3
```

Comment:
This doesn't really have anything to do with the contract... I think it's better to put this in the RPC package.

Reply:
Putting it in RPC package will cause cyclic dependencies, I will put in it config package.